lib/contracts.ts

```diff
@@ -21,16 +21,13 @@ const rinkebyContracts: ContractDirectory = {
 // TODO: @cnasc update to dispatch between test and mainnet based on env
 export const contractDirectory: ContractDirectory = rinkebyContracts;
```

Comment:
do we need to address this TODO in this PR?

Reply:
seems like all refs to `contractDirectory` use rinkebyContracts now

Reply:
at the moment we don't, I forget where the create2 discussion for optimism and polygon left off so not sure on the future disposition

Reply:
sounds good, optimism does have a different contract address, but i can configure this when doing my optimism config PR